src/dotnet-core-uninstall/Shared/Configs/CommandLineConfigs.cs

```diff
@@ -135,6 +135,10 @@ internal static class CommandLineConfigs
             RuntimeInfo.RunningOnWindows ? LocalizableStrings.ForceOptionDescriptionWindows
             : LocalizableStrings.ForceOptionDescriptionMac);
 
+        public static readonly Option MacOSPreserveVSSdksOption = new Option(
+            "--preserve-mac-vs-sdks", 
```

Comment:
I believe adding `mac` in the flags' name is important as to avoid confusion with Windows users (flag is only shown to macOS users, but still)